src/index.js

```diff
@@ -3,6 +3,10 @@ import ReactDOM from 'react-dom';
 import App from './App';
 
 ReactDOM.render(
-  <App />,
+  <App
+    title="title"
+    date="date"
+    text="text"
```

Comment:
no need to pass anything to App, you only need to pass props from App component to Article